README.md

```diff
@@ -37,10 +37,14 @@ addopts =
 ```
 
 #### Without Docker Desktop
-If you're using something like [lima](https://github.com/lima-vm/lima) instead of Docker Desktop, you need to specify a path to Docker API. For lima: 
+If you're using something like [lima](https://github.com/lima-vm/lima) instead of Docker Desktop, you need to specify a path to Docker API. For lima:
 
 `export DOCKER_HOST=unix:///Users/<YOUR-USER>/.lima/default/sock/docker.sock`
 
+If you're using something like [minikube](https://github.com/kubernetes/minikube) instead of Docker Desktop, you need to specify the service host because the exposed ports are not available on localhost. For minikube you get the service IP with `minikube ip` command and then put it to the env var:
```

Comment:
> If you're using something like [minikube](https://github.com/kubernetes/minikube) instead of Docker Desktop, you need to specify the service host 

` If you're using [minikube](https://github.com/kubernetes/minikube), you also need to specify the service host `

> put it to the env var:

`put it in the env var:`

Reply:
Done